build-system/compile/build.conf.js

```diff
@@ -20,13 +20,24 @@ const localPlugin = name =>
   require.resolve(`../babel-plugins/babel-plugin-${name}`);
 
 const defaultPlugins = [
+  '@babel/plugin-transform-react-constant-elements',
+  [
+    '@babel/plugin-transform-react-jsx',
+    {
+      pragma: 'preact.createElement',
```

Comment:
Is there a reason to use `preact.createElement` instead of `h`?

Reply:
The [docs](https://babeljs.io/docs/en/babel-plugin-transform-react-jsx) show it as `Preact` as opposed to `preact`. Does it still work lowercase?

Reply:
> Is there a reason to use preact.createElement instead of h?

`createElement` is self descriptive, and the same as React's.

> The docs show it as Preact as opposed to preact. Does it still work lowercase?

It depends on how we want to import it. I chose

```js
import * as preact from 'preact';
```

Reply:
I'm fine with lowercase, but I just want to point out that I think using uppercase `Preact` / `React` is significantly more common.

Reply:
Done.